staticaddr/deposit/manager.go

```diff
@@ -129,7 +134,11 @@ func (m *Manager) Run(ctx context.Context, initChan chan struct{}) error {
 	for {
 		select {
 		case height := <-newBlockChan:
-			// Inform all active deposits about a new block arrival.
+			// Poll for new deposits and spin up fsms for them.
+			go m.pollDeposits(ctx)
```

Comment:
I'd say the the only blocking call here is `ListUnspent` so instead of spanning on a new goroutine, I'd suggest keeping `pollDeposits` synchronous with a timeout context to guard against `ListUnspent` blocking forever if there's any network/systemic error.

Reply:
the timeout ctx makes sense. Will change this.